sdk/clientcore/core/src/main/java/io/clientcore/core/http/models/AuthMetadata.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package io.clientcore.core.http.models;
+
+import io.clientcore.core.credentials.oauth.OAuthTokenRequestContext;
+
+import java.util.List;
+
+/**
+ * Auth Metadata represents the authentication related data provided at Http Request level to the pipeline policies.
+ */
+public class AuthMetadata {
+    private List<String> authScheme;
```

Comment:
Do we need an enum  instead of String for declaring all the auth schemes?

Reply:
yeah, will make sense to add it.